xonsh/prompt/vc.py

```diff
@@ -149,24 +137,25 @@ def _first_branch_timeout_message():
     )
 
 
+def _has(binary):
```

Comment:
Please name this something more distinct because of amalgamation. Perhaps, `_vc_has`?